infinite/index.ts

```diff
@@ -30,6 +30,7 @@ import type {
 import { useSyncExternalStore } from 'use-sync-external-store/shim/index.js'
 
 const INFINITE_PREFIX = '$inf$'
+const EMPTY_PROMISE = Promise.resolve(UNDEFINED)
```

Comment:
can save 1 byte without passing UNDEFINED?

Reply:
Right, although we need to `as Promise<undefined>` because it will be `Promise<void>` if missing the value.